docs/blog/e18e.md

```diff
@@ -55,16 +55,16 @@ Some examples of these performance optimizations:
 
 - [Speeding up the JavaScript ecosystem](https://marvinh.dev/blog/speeding-up-javascript-ecosystem/)
 - [Astro build performance](https://gist.github.com/bluwy/4eee1ca1ee00cc256d7c769724fe2826)
-- [Typescript package size optimizations](https://x.com/andhaveaniceday/status/1803869279017931187)
+- [TypeScript package size optimizations](https://x.com/andhaveaniceday/status/1803869279017931187)
 and Jake's blog in [similar efforts](https://jakebailey.dev/posts/pnpm-dt-2/)
 
 ## Level up
 
-Many well established packages have grown over time to fit more and more features, configurability and so on. Sometimes, we have use cases where this is too much for what we need, and would rather go back to the core functionality.
+Many well-established packages have grown over time to fit more and more features, configurability and so on. Sometimes, we have use cases where this is too much for what we need, and would rather go back to the core functionality.
 
-In those situations, the community has recently been building more lighter alternatives - packages which often come in with zero dependencies and very small install/bundle sizes.
+In those situations, the community has recently been building more lighter alternatives: packages which often come with zero dependencies and very small install/bundle sizes.
```

Comment:
I don't think we need to change the emdash

Reply:
Now I see that "more lighter" is also not correct. It should be "lighter". I realize it may be futile for me to say so, but please consider that whoever repeatedly wrote "community lead" and many other such mistakes may not have the best judgement on copy editing.